packages/editor/src/components/browser/maximizedCodeEditor/MaximizedCodeEditor.tsx

```diff
@@ -1,19 +1,29 @@
 import './MaximizedCodeEditor.css';
+import type * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
 import { useBrowser, type BrowserType } from '../useBrowser';
 import { useMonacoEditor } from '../../widgets/code-editor/useCodeEditor';
 import CodeEditor from '../../widgets/code-editor/CodeEditor';
 import Browser from '../Browser';
 import { MAXIMIZED_MONACO_OPTIONS } from '../../../monaco/monaco-editor-util';
 
-type MaximizedCodeEditorProps = {
+export type MaximizedCodeEditorProps = {
   editorValue: string;
   location: string;
   browsers: BrowserType[];
   applyEditor: (change: string) => void;
+  cursorPosition: { lineNumber: number; column: number } | null;
+  selectionRange: monaco.IRange | null;
```

Comment:
isn't this the same? if I have a selection the cursor should be at the end of this selection and if the selection is nothing then you have still the position in there (endlinenumber, endcolumn)?